.travis.yml

```diff
@@ -22,20 +21,55 @@ branches:
   only:
     - master
 
+env:
+  global:
```

Comment:
I'd prefer that the jobs always specify `COMPOSER_INSTALL` and `WORDPRESS` so that the reader doesn't have to scroll up and think about what the default is.

Reply:
Eh, actually, it's fine. 🙂

Reply:
I found it was getting pretty messy and repetitive if every job specified the same default values.